app/models/allergy.js

```diff
@@ -2,8 +2,10 @@ import AbstractModel from 'hospitalrun/models/abstract';
 import DS from 'ember-data';
 
 export default AbstractModel.extend({
-  patient: DS.belongsTo('patient'),
+  // Attributes
   name: DS.attr('string'),
   icd9CMCode: DS.attr('string'),
-  icd10Code: DS.attr('string')
+  icd10Code: DS.attr('string'),
+  // Associations
+  patient: DS.belongsTo('patient'),
```

Comment:
No problem, just saw that the test output was a little noisy since I need to clean up a few validators in the tests and wasn't sure if you could find it in there. :tropical_drink: